index.html

```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
     </ul>
 
   </section>
-
+  <section class="contact-form">
+    <form class="my-form" action="https://formspree.io/f/mrgdbqpg" method="POST">
+      <p class="contact-form-content">I'm always interested in hearing about new projects, so if you'd like to chat
+        please get in touch.</p>
+      <input class="contact-form-name" type="text" name="name" placeholder="Full name">
+      <input class="contact-form-email" type="email" name="email" placeholder="Email address">
+      <textarea class="contact-form-message" rows="3" name="message" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Kindly add the validation for the maximum character for this `textarea` to `maxlength="500"`.:wink: